deb-package-builder/build_packages.sh

```diff
@@ -36,5 +36,5 @@ do
     echo "Building packages for PHP ${VERSION}"
     gcloud container builds submit . --config=build-packages.yaml \
                                      --substitutions _PHP_VERSION=${VERSION},_GOOGLE_PROJECT_ID=${GOOGLE_PROJECT_ID},_BUCKET=${BUCKET} \
-                                     --timeout=40m
+                                     --timeout=90m
```

Comment:
If you compile both php and libuv, you will hit 40m. libuv takes ~25 minutes to compile. We can try to optimize this in a separate PR.